### ISSUE TITLE
Upload 20 years of ERA5 data to sampo

### ISSUE THREAD

**Issue by juliasloan25** [CliMA/ClimaLand.jl#902]
**Is your feature request related to a problem? Please describe.**
We need multiple decades of forcing data available for longruns of the full land model. We intend to do these runs on clima, so we can upload the data to sampo.

Part of #859

**Comment by ph-kev** [comment]
Pull request for this in ClimaArtifacts: [https://github.com/CliMA/ClimaArtifacts/pull/63](https://github.com/CliMA/ClimaArtifacts/pull/63)

The current variables in the 2021 ERA5 dataset being used are these:
- Latitude ("lat") 
- Longitude ("lon") 
- Time ("time")
- Temperature at 2 meters (lon, lat, time; "t2m")
- Downwelling SW flux accumulation at surface (lon, lat, time; "ssrd")
- Downwelling LW flux accumulation at surface (lon, lat, time; "strd")
- Neutral wind at 10m, u-component (lon, lat, time; "u10n")
- Neutral wind at 10m, v-component (lon, lat, time; "v10n")
- Surface air pressure (lon, lat, time; "sp")
- Dew point temperature at 2 meters (lon, lat, time; "d2m")
- Total accumulated precipitation (lon, lat, time; "tp")
- Accumulated snowfall (lon, lat, time; "sf")
- Leaf area index, high vegetation (lon, lat, time; "lai_hv")
- Leaf area index, low vegetation (lon, lat, time; "lai_lv")

**Comment by braghiere** [comment]
That is right. Have a look also at `/groups/esm/ClimaArtifacts/artifacts/era5_land_forcing_complete/download_era5_2021.py `

I used that script to get the 2021 data. Can we already replace LAI (low and high) by the MODIS one in this file? In the ILAMB intercomparison, we've noticed ERA5 LAI is too high.

**Comment by ph-kev** [comment]
> That is right. Have a look also at `/groups/esm/ClimaArtifacts/artifacts/era5_land_forcing_complete/download_era5_2021.py `
> 
> I used that script to get the 2021 data. Can we already replace LAI (low and high) by the MODIS one in this file? In the ILAMB intercomparison, we've noticed ERA5 LAI is too high.

Is the MODIS dataset the dataset on ILAMB for Leaf Area Index? I checked that one and the dates only go up to 2005 and the dates are monthly instead of hourly.

**Comment by kmdeck** [comment]
> Pull request for only 2013 in ClimaArtifacts: [CliMA/ClimaArtifacts#63](https://github.com/CliMA/ClimaArtifacts/pull/63)
> 
> The current variables in the 2021 ERA5 dataset being used right now are these:

>     * Downwelling SW flux accumulation at surface (lon, lat, time; "ssrd")
> 
>     * Downwelling LW flux accumulation at surface (lon, lat, time; "strd")
> 
>
> 
>     * Total accumulated precipitation (lon, lat, time; "tp")
> 
>     * Accumulated snowfall (lon, lat, time; "sf")
> 


We can switch from accumulations to means (and alter the preprocessing functions used)

**Comment by kmdeck** [comment]
> That is right. Have a look also at `/groups/esm/ClimaArtifacts/artifacts/era5_land_forcing_complete/download_era5_2021.py `
> 
> I used that script to get the 2021 data. Can we already replace LAI (low and high) by the MODIS one in this file? In the ILAMB intercomparison, we've noticed ERA5 LAI is too high.

Let's still grab the ERA5 LAI data. I updated the issue to note that this can be monthly resolution (and in a different file)

**Comment by Sbozzolo** [comment]
We have the ERA5 data in `/net/sampo/data1/era5`, is anything missing?

**Comment by kmdeck** [comment]
as far as ERA5 goes, yes, we can close this.

Ill open another issue on Monday to add MODIS data to ClimaArtifacts and to switch ClimaLand to that
